app/dashboard/views.py

```diff
@@ -3090,6 +3094,100 @@ def verify_user_google(request):
 
     return redirect('profile_by_tab', 'trust')
 
+
+@login_required
+def verify_profile_with_ens(request):
+    profile = request.user.profile
+    user_address = profile.preferred_payout_address
+
+    # 1. Check user has preferred address
+    if not is_address(user_address):
+        return JsonResponse({
+            'error': 'NO_PAYOUT_ADDRESS_ASSOCIATED',
+            'msg': 'You don\'t have one preferred payout address associated to your profile',
+            'data': {
+                'step': 1,
+                'verified': profile.is_ens_verified
+            }
+        })
+
+    # 2. Check Reverse lookup
+    web3 = get_web3('mainnet')
+    ns = ens.ENS.fromWeb3(web3, '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e')
+    node = ns.name(user_address)
+
+    if not node:
+        return JsonResponse({
+            'error': 'NO_ENS_NAME_ASSOCIATED',
+            'msg': f'You don\'t have one ens domain associated to your address {user_address}',
+            'data': {
+                'step': 2,
+                'verified': profile.is_ens_verified
+            }
+        })
+
+    # 3. Check Forward lookup
+    registered_address = ns.address(node)
+    if not is_address(registered_address):
+        return JsonResponse({
+            'error': 'NO_ADDRESS_ASSOCIATED_TO_ENS',
+            'msg': f'You don\'t have associated your address to your domain {node}',
```

Comment:
Can you change this to `'You don\'t have associated your address to your domain {node}'. Please add your ETH address at https://app.ens.domains/name/{node}`